Noemi_first_try/questions/angst.html

```diff
@@ -0,0 +1,19 @@
+<html>
+  <head>
+    <link rel="stylesheet" href="../index/style.css">
+  </head>
+  <body>
+    <header><h1>Angst</h1></header>
```

Comment:
Here for example you can use a header class name or id and then use a different background-image url in CSS.